### PR TITLE
feat: documenting sessions

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Remove expiring app docs
- Add session docs
 

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -261,6 +261,7 @@ const sidebars = {
         "api/tokens/token-associations",
       ],
     },
+    "api/sessions",
```
Comment (bweber) [inline sidebars.js:264]: This should be nested under applications nav item.

**Review by bweber — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 1 above)
Comment (dhudec) [reply sidebars.js:264]: But why? The api routes for sessions are not nested under applications and sessions are separate from applications. I feel like if we want to group them together we should change the top level from Applications to something like Access Controls and nest Sessions, Applications, and Permissions all underneath

**Comment by dhudec** [comment]
GH won't let me comment on an unchanged line.. so here:

[This line](https://github.com/Basis-Theory/developers.basistheory.com/pull/31/files#diff-46234ba39f3f53b6ccf7926341e3aea32720ce35f2fa761d1b01fce9f942dff8R17) should probably change to say

>Your ability to authenticate to the Basis Theory API is granted through an API Key associated with an Application or Session.

File: sidebars.js [hunk 2]
```diff
@@ -217,7 +217,10 @@ const sidebars = {
         type: "doc",
         id: "api/applications/applications",
       },
-      items: ["api/applications/permissions"],
+      items: [
+        "api/applications/permissions",
+        "api/sessions"
```
Comment (bweber) [inline sidebars.js:222]: Can we move the doc under the applications folder also so grouping matches?

**Review by bweber — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 2 above)
Comment (jleon15) [reply sidebars.js:222]: no

**Review by bweber — APPROVED** [review]

**Pull request merged**